be/src/olap/tablet.h

```diff
@@ -708,6 +709,13 @@ inline void Tablet::set_cumulative_layer_point(int64_t new_point) {
     _cumulative_point = new_point;
 }
 
+inline void Tablet::save_cumulative_layer_point(int64_t new_point) {
```

Comment:
Is there any difference between set_cumulative_layer_point? I only find some checks.